gcsio/src/main/java/com/google/cloud/hadoop/gcsio/authorization/AuthorizationHandler.java

```diff
@@ -27,17 +29,19 @@
    *
    * @param resource A GCS object URI.
    * @throws AccessDeniedException Thrown when access denied.
+   * @return Access Token.
    */
-  void handleListObjects(URI resource) throws AccessDeniedException;
+  String handleListObjects(URI resource) throws AccessDeniedException;
```

Comment:
Updating the public method signature would be backwards in-compatible. Do we want to introduce default methods for the existing APIs to maintain compatibility ?